pom.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2020, RTE (http://www.rte-france.com)
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-->
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.gridsuite</groupId>
```

Comment:
Use powsybl parent pom

Reply:
I tried to use the powsybl parent pom, but now i got an error with checkstyle on the NetworkModificationApplication class : it says that this class is a utility class, and so it should not have a default constructor. So i try to add a private default constructor, but now it says that the class should be final, but it is not accepted at execution ....

Is there a lombok annotation to avoid this error with checkstyle ?

Reply:
There is a way to discard this checkstyle error with a comment. You can have a look at the other web services for that.

Reply:
It seems that the other web services currently don't use the powsybl parent pom, and in their own checkstyle.xml file, they disactivate the 'HideUtilityClassConstructor' check, but this check is present in the powsybl parent checkstyle xml file

So, should i keep a checkstyle xml file in the new modification-server service, or else do you know if there is a way to override the parent checkstyle configuration file, in order to suppress some checks ?

Reply:
Not all webservices have been migrated to the new parent pom :(
This one has been migrated: https://github.com/powsybl/powsybl-network-store/blob/6cebe249cb6d2d56224c3a30d2afa70ef22c1591/network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreApplication.java#L6

Reply:
Done